src/components/TypeBox/index.tsx

```diff
@@ -4,20 +4,21 @@ import {useRouter} from "next/navigation";
 import React, {useEffect, useRef, useState} from "react";
 import {useTranslation} from "react-i18next";
 
-import {WORDS_BY_LENGTH} from "src/utils/constants";
-import {DIFFICULTY_SETTINGS} from "src/utils/difficulty";
 import {GameOverData} from "src/models/gameOverData";
+import enDataset from "src/datasets/en.json";
+import esDataset from "src/datasets/es.json";
+import {DIFFICULTY_SETTINGS} from "src/utils/difficulty";
 import {calculateScore} from "src/utils/score";
 
-import {TypeBoxProps} from "./types";
+import {DatasetLanguage, TypeBoxProps, WordsDataset} from "./types";
 
 const TypeBox = ({
   difficulty = "normal",
   initialTime,
   wordsQuantity,
+  datasetLanguage = "en",
```

Comment:
Aca podria utilizarse directamente esto ?
```suggestion
  datasetLanguage = LANGUAGE.EN,
```